test.sh

```diff
@@ -23,7 +23,7 @@ fi
 
 if [ -z "$registry_id" ]; then
     registry_id=$prefix-registry
-    docker run -d -p 5000 --name $registry_id registry:2 >/dev/null
+    docker run -d -p 5000:5000 --name $registry_id registry:2 >/dev/null
```

Comment:
Do we lose something by having a fixed port here? I don't know why the dynamic port isn't working and the fixed port is. Maybe needs more investigation?

Reply:
I don't think we lose too much the only thing I see if the port is already in use by other app, I could not make it work without the port